src/toxcore_tests/dht_tests.rs

```diff
@@ -816,7 +817,7 @@ fn get_nodes_to_bytes_test() {
         let g_bytes = gn.to_bytes();
         let PublicKey(pk_bytes) = gn.pk;
         assert_eq!(&pk_bytes, &g_bytes[..PUBLICKEYBYTES]);
-        assert_eq!(&u64_to_array(gn.id), &g_bytes[PUBLICKEYBYTES..]);
+        assert_eq!(gn.id, LittleEndian::read_u64(&g_bytes[PUBLICKEYBYTES..]));
```

Comment:
>@zetok are you ok with such changes?

Well, when there's no endianess specified, `NativeEndian` most likely should be used.

>And completely remove xx_to_array.

Since `byteorder` crate provides the functionality, I don't see any problem with removing it.